src/LoginRouter.js

```diff
@@ -154,7 +154,7 @@ function (BaseLoginRouter,
     ],
 
     defaultAuth: function() {
-      if(this.settings.get('features.idpDiscovery')) {
+      if(this.settings.get('features.idpDiscovery') || this.settings.get('features.identifierFirst')) {
```

Comment:
Do we still need this?

Reply:
ah, good point. We do not need it, and probably removing it will also enforce better the dependency. Will remove.